src/plugins/auto/plugin.cpp

```diff
@@ -130,33 +134,60 @@ std::vector<DeviceInformation> MultiDeviceInferencePlugin::ParseMetaDevices(cons
             }
         }
 
-        std::string defaultDeviceID = "";
         DeviceIDParser parsed{deviceName};
         std::string deviceid = parsed.getDeviceID();
-        if (deviceid.empty()) {
-            defaultDeviceID = getDefaultDeviceID(deviceName);
-            deviceid = defaultDeviceID;
+        std::vector<std::string> sameTypeDevices;
+        // if AUTO:GPU case, replace GPU with GPU.0 and GPU.1
+        // Disable AUTO:MYRIAD here because of below test case
+        // MYRIAD/CoreThreadingTests.smoke_QueryNetwork/targetDevice=MULTI_config=MULTI_DEVICE_PRIORITIES:MYRIAD_
+        // faild on windows
+        // the error is
+        // myriadFuncTests-0 INFO: [E:] [BSL] found 0 ioexpander device
```

Comment:
this maybe particular issue of the CI, not myriad itself, adding @evgenytalanin-intel